Replicator/Replicator.cc

```diff
@@ -469,7 +469,8 @@ namespace litecore { namespace repl {
                 gotError(error);
                 stop();
                 return false;
-            } else if (_options.properties[kC4ReplicatorResetCheckpoint].asBool()) {
+            } else if (reset) {
+                _checkpointer.reset();
```

Comment:
Doesn't the checkpoint has already been reset as of line 461 when calling _checkpointer.read() with reset = true?

Reply:
Oh right good catch.  This is leftover from a previous failed way of doing it.